### ISSUE TITLE
Add growing degree days (GDD) to trial metadata options

### ISSUE THREAD

**Issue by bellerbrock** [solgenomics/sgn#3371]
Expected Behavior 
--------------------------------------------------------------------------

This came up in this morning's Sweetgains WP3 meeting. It's not always practical to analyze time-series weather data, but derived values like GDD or total precipitation that are just a single measurement per trial can be useful and are maybe more realistic to store in Breedbase. 

There is already [a closed issue](https://github.com/solgenomics/sgn/issues/2610) for adding GDD functionality, but from what I can tell it's specific to drone imaging and/or U.S. locations that have a NOAA weather station nearby. So we'd need something else for breeders who are working in other countries and/or deriving GDD values from their own weather data.

For Bugs:
---------

**Comment by nickmorales** [comment]
Right, there is a function here https://github.com/solgenomics/sgn/blob/master/lib/CXGN/NOAANCDC.pm to calculate growing degree days or cumulative precipitation using the get_temperature_averaged_gdd() or get_averaged_precipitation() functions. It uses the NOAA station id that is saved in breedbase for a location.
